src/store/modules/inbox.js

```diff
@@ -2,82 +2,23 @@
 // import InboxService from '../../services/inbox/inbox';
 
 const state = {
-  inbox: [
-        {isUser: false, text: 'Oh... Well, this is akward.'},
-        {isUser: true, text: 'I meant I am working too hard; I am all fired up right now.'},
-        {isUser: false, text: 'I appreciate your concern. Apply water vigorously and call 911.'},
-        {isUser: true, text: 'I need help! My desk is on fire!'},
-        {isUser: false, text: 'Hi, how can I help you?'},
-        {isUser: false, text: 'Oh... Well, this is akward.'},
-        {isUser: true, text: 'I meant I am working too hard; I am all fired up right now.'},
-        {isUser: false, text: 'I appreciate your concern. Apply water vigorously and call 911.'},
-        {isUser: true, text: 'I need help! My desk is on fire!'},
-        {isUser: false, text: 'Hi, how can I help you?'}]
-}
+  selectedInboxItemId: 0,
```

Comment:
By default we will want to have this set to VC chatbot but in a future PR

Reply:
Correct, that will be part of whoever is doing the inbox functionality PR.